delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/syntax/ProjectionErrorsSyntax.scala

```diff
@@ -30,35 +22,8 @@ trait ProjectionErrorsSyntax {
 
 object ProjectionErrorsSyntax {
 
-  implicit private val api: JsonLdApi = JsonLdJavaApi.lenient
-  private val defaultPrinter: Printer = Printer(dropNullValues = true, indent = "")
-
   final class ProjectionErrorsOps(val projectionErrors: ProjectionErrors) extends AnyVal {
 
-    /**
-      * Get available failed elem entries for a given projection (provided by project and id), starting from a failed
-      * elem offset as a stream of Server Sent Events
-      *
-      * @param projectionProject
-      *   the project the projection belongs to
-      * @param projectionId
-      *   IRI of the projection
-      * @param offset
-      *   failed elem offset
-      */
-    def sses(projectionProject: ProjectRef, projectionId: Iri, offset: Offset)(implicit
-        rcr: RemoteContextResolution
-    ): ServerSentEventStream =
-      projectionErrors.failedElemEntries(projectionProject, projectionId, offset).evalMap { felem =>
```

Comment:
Is the method on projection errors used somewhere else ?
It may be dead code

Reply:
Indeed